e2e-tests/package.json

```diff
@@ -42,7 +42,7 @@
     "node": "20.* || 22.*"
   },
   "lint-staged": {
-    "*.js": "eslint --fix",
-    "*.{js,scss,yaml}": "prettier --write"
+    "*.js": ["eslint --fix", "prettier --write"],
```

Comment:
We might want to propagate this change to all our `lint-staged` setups right now since they could also have the same issue